PROFILE.md

```diff
@@ -0,0 +1 @@
+Welcome to my GitHub profile!
```

Comment:
Hello there, my name is Skyler and I am very eager the world the world of coding. Looking forward to what I can learn and contribute.